<launch>

  <node name="F1_undistort_node" pkg="image_undistort" type="image_undistort_node">
    <param name="input_camera_namespace" value="F1"/>      
    <param name="input_camera_info_from_ros_params" value = "true"/>    
    <param name="scale" value="1.0"/>
    <param name="output_camera_info_source" value="auto_generated"/>
    <rosparam file="$(find localization)/config/calibrations/ig2/F1.yaml"/>
    <remap from="input/image" to="F1/image_mono"/>
    <remap from="output/image" to="F1/image_rect"/>
  </node>

  <node pkg="localization" type="search_images.py" name="search_images" >
    <remap from="image" to= "/F1/image_rect" />
  </node>

</launch>
